proxy.go

```diff
@@ -518,12 +518,30 @@ func (rp *reverseProxy) getScope(req *http.Request) (*scope, int, error) {
 		c = rp.clusters[u.toCluster]
 		cu = c.users[u.toUser]
 	}
+
+	isTransparent := false
+	if u == nil {
+		u = rp.users[config.TransparentUser]
+		if u != nil {
+			c = rp.clusters[u.toCluster]
+			if c != nil {
+				transparent_cu := *c.users[u.toUser]
```

Comment:
Let's check if c.users[u.toUser] is not nil

Reply:
This is already checked, see comment above that says "c and cu for toCluster and toUser must exist if applyConfig is correct. Fix applyConfig if c or cu equal to nil."

Actually I would have added an assert if I knew that a concept of assert exists in golang ;)  Looking into this.